modular_skyrat/modules/customization/modules/mob/living/carbon/human/species.dm

```diff
@@ -133,6 +133,16 @@ GLOBAL_LIST_EMPTY(customizable_races)
 					underwear_overlay.color = species_human.underwear_color
 				standing += underwear_overlay
 
+		if(species_human.bra && !(species_human.underwear_visibility & UNDERWEAR_HIDE_BRA))
+			var/datum/sprite_accessory/bra/bra = GLOB.bra_list[species_human.bra]
+			var/mutable_appearance/bra_overlay
+			if(bra)
```

Comment:
No need to have it before the `if`, it's just there for nothing if it doesn't go through the condition.
```suggestion

			if(bra)
					var/mutable_appearance/bra_overlay
```